### ISSUE TITLE
Unhandled Rejection in NFT example 

### ISSUE THREAD

**Issue by leopk-code** [austintgriffith/scaffold-eth#308]
I am getting "Unhandled Rejection (Error): call revert exception (method="getReserves()", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.3.0)" when running the NFT example. This occurs even before minting any Tokens, when deployed to the Localhost:3000.

I am using Metamask 

![image](https://user-images.githubusercontent.com/67122519/128633520-25b67eda-5ee6-4989-8453-39f54b046717.png)

Are there any known remedies for this issue?

**Comment by genekogan** [comment]
I'm also getting this error. Appears to be coming from `useExchangePrice` hook.

**Comment by futurethang** [comment]
maybe not a long term solution, but this somewhat patches it enough to tinker:
```javascript
const pair = await Fetcher.fetchPairData(DAI, WETH[DAI.chainId], mainnetProvider).catch(e => {
          console.error(e);
        });
        if (pair !== undefined) {
          const route = new Route([pair], WETH[DAI.chainId]);
          setPrice(parseFloat(route.midPrice.toSignificant(6)));
        }
```

**Comment by codenamejason** [comment]
Has this been resolved @futurethang

**Comment by futurethang** [comment]
@codenamejason  assuming a fix didn't come from another direction, PR #311 is open for any feedback. Are you still experiencing the bug yourself?
